.gitattributes

```diff
@@ -0,0 +1,2 @@
+u-boot/rockpi4/m0/rk3399m0.bin filter=lfs diff=lfs merge=lfs -text
+u-boot/rockpi4/m0/rk3399m0pmu.bin filter=lfs diff=lfs merge=lfs -text
```

Comment:
I think that can be just `*.bin` if we agree to use `bin` extension for binary files only and always use Git LFS for them

Reply:
sure, updated